index.test.js

```diff
@@ -571,7 +570,6 @@ describe('Configure AWS Credentials', () => {
     test('only role arn and region provided to use GH OIDC Token', async () => {
         process.env.ACTIONS_ID_TOKEN_REQUEST_TOKEN = 'test-token';
```

Comment:
Do we need these environment variables anymore?

Reply:
Yes. These being set is how we determine whether or not we are in a self-hosted runner or in a "proper" GitHub Action. If we remove them, the test will behave as if it were in a self-hosted runner (where OIDC isn't supported)

Reply:
Then where are we using it's **test-token** value in the test?

Reply:
We don't use it in the test, we use it in the action.

https://github.com/aws-actions/configure-aws-credentials/blob/master/index.js#L284

Reply:
We're less concerned with the actual value of the ENV variable, we just need it set to something so that the action sees that it has been set and behaves as if the test were running in a 'real' GitHub Action.